src/main/java/net/syscon/elite/api/model/CourtHearings.java

```diff
@@ -0,0 +1,17 @@
+package net.syscon.elite.api.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import io.swagger.annotations.ApiModel;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
```

Comment:
You'll probably need this NoArgsConstructor?

Reply:
Ah I did have but whilst it was empty if refused to compile ;)